roadmap.md

```diff
@@ -14,8 +14,7 @@ The roadmap below communicates priorities for evolving and extending the scope o
 |1  |Incorporating .NET Framework servicing fixes into .NET Core 3.1, .NET 5 and .NET 6 |Ongoing|
 |2  |Performance Improvements |Ongoing|
 |3  |Ongoing support and maintenance for .NET Framework, .NET core 3.1, .NET 5.0 and .NET 6.0 |Ongoing|
-|4  |New API review process for Community PRs  |21H2|
```

Comment:
Is this removal right ? Does this mean that the WPF team does not plan to accept PRs from the community to add new APIs in the near future ?

Reply:
Hi @ThomasGoulet73, this has been updated correctly. Thanks!